crates/cdk/src/nuts/nut00/mod.rs

```diff
@@ -374,45 +374,69 @@ pub enum CurrencyUnit {
     /// Euro
     Eur,
     /// Custom currency unit
-    Custom(String),
+    Custom(String, u32),
```

Comment:
Do you mean in the ecash token? Is this a problem?

Reply:
Since the serialize/deserialize implementation just use to/from string internally and this modifies those to include or expect the derivation path when [the token](https://github.com/cashubtc/cdk/blob/5dd196d8f580a4b4753c4523c7b52847237e587f/crates/cdk/src/nuts/nut00/token.rs#L284-L300) (cashuB...) get serialized/desrialized it would include or expect the path.